profiles/models_test.py

```diff
@@ -37,16 +46,36 @@ def test_student_id_doesnt_change_on_save(self):
         profile.refresh_from_db()
         assert profile.student_id == current_id
 
-    def test_student_id_should_pretty_print(self):
+    def test_should_pretty_print(self):
         """test pretty printing property method"""
         with mute_signals(post_save):
             profile = ProfileFactory()
         assert profile.pretty_printed_student_id == "MMM{0:06}".format(profile.student_id)
 
-    def test_student_id_should_be_blank_if_not_yet_saved(self):
+    def test_should_be_blank_if_not_yet_saved(self):
         """test student id when model instance not yet saved"""
         with mute_signals(post_save):
             user = UserFactory()
         profile = Profile(user=user)
         assert profile.student_id is None
         assert profile.pretty_printed_student_id is ''
+
+
+class ImageTests(ESTestCase):
+    """
+    Tests for image fields
+    """
+    def test_upload_to(self):
+        """
+        Image upload_to should have a function which creates a path
+        """
+        # pin the timestamps used in creating the URL
+        with patch('profiles.util.datetime', autospec=True) as mocked_datetime:
+            mocked_datetime.now = Mock(return_value=datetime.now(tz=pytz.UTC))
```

Comment:
you do not need to do this: by default the `new_callable` in a `patch` is a `MagicMock`, so you can simply define the return value for the function you want.
`mocked_datetime.now.return_value = datetime.now(tz=pytz.UTC)`